CLEANUP_MODULES

```diff
@@ -0,0 +1,11 @@
+tests/console/ttylogin4.pm
+tests/console/yast2_nis.pm
+tests/installation/inst_mediacheck.pm
+tests/x11/mate_reconfigure_openqa.pm
```

Comment:
the reconfigure_openqa are obsolete - we handle other desktops differently meanwhile